examples/huggingface/pytorch/text-generation/quantization/llm_quantization_recipes.md

```diff
@@ -0,0 +1,212 @@
+## Step-by-Step recipes for LLM quantization
+
+This document describes the step-by-step instructions to run large language models (LLMs) on 4th Gen Intel® Xeon® Scalable Processor (codenamed Sapphire Rapids) with [PyTorch](https://pytorch.org/) and [Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch).
+
+The scripts [run_generation.py](./run_generation.py) provide two quantization approaches respectively (SmoothQuant, Weight-Only Quantization) based on [Intel® Neural Compressor](https://github.com/intel/neural-compressor) and return last token prediction accuracy by `trainer`.
+
```

Comment:
List the model here, we will have more and more models in the feature, make sure people could find the model they wanted quickly.